FinalXmL/FinalXmL/Geonamedata.cs

```diff
@@ -8,7 +8,7 @@
 
 
 
-namespace QuickType1
+namespace QuickTypeGeo
 {
```

Comment:
QuickType1 is too vague of a name

Reply:
Exactly, yes, good point.   While you're in there, I'd rename it to something more descriptive than "QuickTypeGeo" as well.